lib/widgets/recent_dm_conversations.dart

```diff
@@ -48,8 +51,11 @@ class _RecentDmConversationsPageBodyState extends State<RecentDmConversationsPag
 
   @override
   Widget build(BuildContext context) {
+    final designVariables = DesignVariables.of(context);
+    final zulipLocalizations = ZulipLocalizations.of(context);
     final sorted = model!.sorted;
-    return SafeArea(
+    return Scaffold(
+      body: SafeArea(
```

Comment:
This will mean a `Scaffold` nested inside a `Scaffold`. See the Scaffold docs on that:
https://main-api.flutter.dev/flutter/material/Scaffold-class.html#nested-scaffolds

Have you tried alternatives to nesting the Scaffolds?